sklearn/svm/_classes.py

```diff
@@ -1086,6 +1086,9 @@ class NuSVC(BaseSVC):
     Pipeline(steps=[('standardscaler', StandardScaler()), ('nusvc', NuSVC())])
     >>> print(clf.predict([[-0.8, -1]]))
     [1]
+
+    See :ref:`sphx_glr_auto_examples_svm_plot_svm_nonlinear.py`
+    for a more detailed example.
```

Comment:
I think `plot_svm_nonlinear.py` could also be removed, we have the other example that @StefanieSenger is working on which nicely includes non linear kernels. WDYT @glemaitre

Reply:
It is true. Here, this is the typical XOR example. We should probably have this examples for all classifiers but not specifically for the SVM.